src/webgpu/api/validation/resource_usages/texture/in_pass_encoder.spec.ts

```diff
@@ -1030,6 +1032,14 @@ g.test('bindings_in_bundle')
   });
 
 g.test('unused_bindings_in_pipeline')
+  .desc(
+    `
+    Test the texture usages with unused bindings in the pipeline.
+      - Test the recording succeeds in the compute pass even though it's bindings is not used in the
+        pipeline.
+      - Test the recording fails in the render pass.
```

Comment:
```suggestion
    Test that for compute pipelines with 'auto' layout, only bindings used by the pipeline count
    toward the usage scope. For render passes, test the pipeline doesn't matter because only the
    calls to setBindGroup count toward the usage scope.
```